lesson_19/tests/test_add_player_page.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+
+@pytest.mark.regression
+def test_add_player_with_required_fields(open_add_player_page):
+    assert open_add_player_page.is_required_fields_empty(), "Required fields are not empty before sending data"
+    text = open_add_player_page.add_player_with_required_fields().wait_for_add_player_confirmation_popup()
+    assert text == 'Added player.' or text == 'Dodano gracza.'
+
+
+@pytest.mark.regression
+def test_add_player_without_age(open_add_player_page):
+    try:
+        open_add_player_page.add_player_without_age()
+        assert False, "Confirmation popup is displayed when it shouldn't be"
+    except TimeoutException:
+        assert True
+
+
+# Code does not work correct: input field is cleared, but then instead of sending next email value from the list,
+# input is a concatenated value of previous and next values
```

Comment:
did you try js to send keys to this input?

Reply:
Thank you very match, it works when I use send keys via JS!